src/components/Chat/Playback/playback.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface PlaybackAttachmentType {
+  title: string;
+  index: string | number;
+}
+
+export interface ActiveMessage {
```

Comment:
Why existing message type is not suitable here?

Reply:
Use existing message type.
File removed as redundant.